packages/amplify_core/lib/src/io/aws_file_platform_html.dart

```diff
@@ -88,45 +89,50 @@ class AWSFilePlatform extends AWSFile {
   @override
   String? get contentType => _contentType ?? super.contentType;
 
-  Future<Blob> get _resolvedBlob async {
-    final resolvedBlobFromPath = _resolvedBlobFromPath;
-    if (resolvedBlobFromPath != null) {
-      return resolvedBlobFromPath;
-    }
-
-    final resolvedBlob = await _resolveBlobFromPath();
-    _size = resolvedBlob.size;
-    _resolvedBlobFromPath = resolvedBlob;
-
-    return resolvedBlob;
-  }
-
-  /// {@macro amplify_core.io.aws_file.chunked_reader}
   @override
-  ChunkedStreamReader<int> getChunkedStreamReader() {
+  Stream<List<int>> get stream {
     final file = _inputFile ?? _inputBlob;
     if (file != null) {
-      return ChunkedStreamReader(_getReadStream(file));
+      return _getReadStream(file);
     }
 
     final inputStream = super.stream;
     if (inputStream != null) {
-      return ChunkedStreamReader(inputStream);
+      return inputStream;
     }
 
     final inputBytes = super.bytes;
     if (inputBytes != null) {
-      return ChunkedStreamReader(Stream.value(inputBytes));
+      return Stream.value(inputBytes);
     }
 
     final path = super.path;
     if (path != null) {
-      return ChunkedStreamReader(_getReadStream(_resolvedBlob));
+      return _getReadStream(_resolvedBlob);
     }
 
     throw const InvalidFileException();
   }
 
+  Future<Blob> get _resolvedBlob async {
+    final resolvedBlobFromPath = _resolvedBlobFromPath;
+    if (resolvedBlobFromPath != null) {
+      return resolvedBlobFromPath;
+    }
+
+    final resolvedBlob = await _resolveBlobFromPath();
+    _size = resolvedBlob.size;
+    _resolvedBlobFromPath = resolvedBlob;
+
+    return resolvedBlob;
+  }
+
+  /// {@macro amplify_core.io.aws_file.chunked_reader}
```

Comment:
Don't need a doc comment when overriding unless you want to override the doc comment since it will inherit the doc comments from the super member